### PR TITLE
[linuxinput] handle keys not known by libevdev

### PR DESCRIPTION
**Pull request by t-8ch**
As reported in https://community.openhab.org/t/linuxinput-binding-and-mouse-capture/122612/8 the binding would not properly handle channels for keys unknown by libevdev.
Fall back to the numeric event codes in this case.

### CODE REVIEW

File: bundles/org.openhab.binding.linuxinput/README.md [hunk 1]
```diff
@@ -81,3 +81,13 @@ The following happens when pressing and releasing a key:
...
+
+#### Channel names
+
+The binding tries to translate the numeric event codes to their symbolic names.
+`KEY_1`, `KEY_A`, `KEY_BACKSPACE` etc.
+
+If the currently installed version of libevdev does not know the symbolic name
+of a key, the numeric value is used.
+Please note that future versions of libevdev may start translating the symbolic
+names.
```
Comment (lolodomo) [inline bundles/org.openhab.binding.linuxinput/README.md:93]: Please do not cut a sentence on several lines.
One sentence = one line.
Two sentences = two lines.

File: bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/LinuxInputHandler.java [hunk 2]
```diff
@@ -95,9 +95,13 @@ boolean delayedSetup() throws IOException {
         EvdevDevice newDevice = new EvdevDevice(config.path);
         for (EvdevDevice.Key o : newDevice.enumerateKeys()) {
             String name = o.getName();
+            if (name == null) {
+                name = Integer.toString(o.getCode());
```
Comment (lolodomo) [inline bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/LinuxInputHandler.java:99]: Are you certain that `o.getCode()` always represent an integer ?
Catching the exception could be more secure.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.linuxinput/README.md (hunk 1 above)
Comment (t-8ch) [reply bundles/org.openhab.binding.linuxinput/README.md]: Done

File: bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/LinuxInputHandler.java (hunk 2 above)
Comment (t-8ch) [reply bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/LinuxInputHandler.java:99]: I'm not sure I understand.
Integers are how all events are represented by the kernel and libevdev.

`getCode()` is guaranteed to return an integer:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.linuxinput/src/main/java/org/openhab/binding/linuxinput/internal/evdev4j/EvdevDevice.java#L211

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by t-8ch** [comment]
Thanks!

**Comment by t-8ch** [comment]
3.3 backport: (remove `.zip` suffix) 
[org.openhab.binding.linuxinput-3.3.0.jar.zip](https://github.com/openhab/openhab-addons/files/9910187/org.openhab.binding.linuxinput-3.3.0.jar.zip)

**Pull request merged**